### PR TITLE
PLANET-5263: Decrease H1 size on mobile 

### PR DESCRIPTION
**Pull request by pablocubico**
Ref: https://jira.greenpeace.org/browse/PLANET-5263

### CODE REVIEW

File: src/base/_typography.scss [hunk 1]
```diff
@@ -53,7 +53,7 @@ h6 {
 h1 {
   line-height: 1.225;
   margin-bottom: 30px;
-  font-size: $font-size-xxxl;
+  font-size: $font-size-xxl;
 
   @include medium-and-up {
     font-size: $font-size-xxxxl;
```
Comment (mleray) [inline src/base/_typography.scss:59]: Do we still want to use `xxxxl` instead of `xxxl` on medium screens and up? Isn't it too big of a jump? 🤔

Comment (pablocubico) [reply src/base/_typography.scss:59]: Hmmm not sure, I was just following the specs from the ticket, I'll leave a comment in JIRA with your question for Will.

https://jira.greenpeace.org/browse/PLANET-5263

Comment (mleray) [reply src/base/_typography.scss:59]: Will is currently reviewing all the heading rules for this ticket: https://jira.greenpeace.org/browse/PLANET-5295

Comment (mleray) [reply src/base/_typography.scss:59]: Okay yeah Will approved 👍

**Pull request merged**